src/utils/supports.js

```diff
@@ -25,11 +26,19 @@ const platformFeatures = {
   '@pleroma/local-visibility-post': containPleroma,
   '@akkoma/local-visibility-post': containAkkoma,
 };
+
 const supportsCache = {};
 
 function supports(feature) {
   try {
-    const { version, domain } = getCurrentInstance();
+    let { version, domain } = getCurrentInstance();
+    let softwareName = getCurrentNodeInfo()?.software?.name || 'mastodon';
+
+    if (softwareName === 'hometown') {
+      // Hometown is a Mastodon fork and inherits its features
```

Comment:
Wondering, how is this relevant to the current PR, which is meant for GTS?

Reply:
When a future Hometown release includes the fix for nodeInfo, `softwareName` will be `'hometown'`, so for features like `'@mastodon/list-exclusive'`, this comparison (`'mastodon' === 'hometown'`) will be false:

```.js
const doesSoftwareMatch = featureSoftware === softwareName.toLowerCase();
```